Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/PaginationPanel.java

```diff
@@ -374,6 +374,8 @@ public void startPaginationClick() {
                 mediaUnits.get(i).setOrderlabel(paginator.next());
             }
         }
+        paginationSelectionSelectedItems = new ArrayList<>();
```

Comment:
This is already done in `preparePaginationSelectionItems` isn't it?

Reply:
Yes, but `preparePaginationSelectionItems` is not always called directly before a pagination because the user can make more than one pagination after each other before closing the popup. And if this PR https://github.com/kitodo/kitodo-production/pull/4214 will be merged `preparePaginationSelectionItems` will be called only one time by opening the metadata editor.

Reply:
Oh I was confued between `paginationSelectionSelectedItems `and `paginationSelectionItems ` 🤦